.github/workflows/release.yml

```
@@ -0,0 +1,175 @@
+          fetch-depth: 0
+
+      - name: 🐹 Set up Go
+        uses: actions/setup-go@v6
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: true
+          cache-dependency-path: api/go.sum
+
+      - name: 🏷️ Extract version
+        id: version
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:7:30: Double quote to prevent globbing and word splitting

(shellcheck)